test/e2e/SKIPPED_TESTS.yaml

```diff
@@ -67,6 +67,7 @@ skipped_tests:
 - TestTinkerbellKubernetes123RedHatSimpleFlow
 - TestTinkerbellKubernetes124RedHatSimpleFlow
 - TestTinkerbellKubernetes125RedHatSimpleFlow
+- TestTinkerbellKubernetes126RedHatSimpleFlow
```

Comment:
Why are we skipping this one?

Reply:
I added this to skipped tests for now because of the CI hardware limitations. It should probably be fine to not skip this test, but I wanted to make sure the 1-27 test would have a good run